src/main/java/com/netgrif/application/engine/petrinet/web/PetriNetController.java

```diff
@@ -79,15 +84,15 @@ public class PetriNetController {
     })
     @RequestMapping(value = "/import", method = POST, produces = MediaTypes.HAL_JSON_VALUE)
     public EntityModel<EventOutcomeWithMessage> importPetriNet(
-            @RequestParam(value = "file", required = true) MultipartFile multipartFile,
+            @RequestParam(value = "file") MultipartFile multipartFile,
```

Comment:
why is file no longer required?

Reply:
It is still required. But the `required` is true by default.